swift/swift.go

```diff
@@ -111,14 +116,19 @@ func (c *Client) GetReader(containerName, objectName string) (io.ReadCloser, htt
 }
 
 // GetReadSeeker creates a read seeker closer to the specified object's data.
-func (c *Client) GetReadSeeker(containerName, objectName string) (io.ReadSeeker, http.Header, error) {
+func (c *Client) GetReadSeeker(containerName, objectName string) (ReadSeekCloser, http.Header, error) {
```

Comment:
In a subsequent PR, I'm planning to remove GetReadSeeker (it's new and only used in one place) and add OpenObject that returns a ReadSeekCloser, and make GetReader use that.
Alternatively, I could change the signature of GetReader but I'd prefer not to break backward compatibility for a method that's been around for a long time.

(there is already a GetReader method)